src/main/java/de/tum/in/www1/artemis/domain/LegalDocumentType.java

```diff
@@ -0,0 +1,13 @@
+package de.tum.in.www1.artemis.domain;
+
+public enum LegalDocumentType {
+
+    PRIVACY_STATEMENT, IMPRINT;
+
+    public String getFileBaseName() {
+        return switch (this) {
+            case PRIVACY_STATEMENT -> "privacy_statement_";
+            case IMPRINT -> "imprint_";
+        };
+    }
+}
```

Comment:
As Teamscale says, some comments would be nice here

Reply:
Also, why do you need this enum? Isn't just pure inheritance enough? Your getFilenameBaseName could also be overwritten based on the class

Reply:
it's convenient to use the enum e.g. for the get requests becuase there's no need to pass a complete document object. 
I added the comments